From 70e184c0616db4e32ea0e4f57c03027b35527fad Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Tue, 4 Jul 2017 14:51:52 +0900
Subject: [PATCH 2550/2848] HDFS-12078. Add time unit to the description of
 property dfs.namenode.stale.datanode.interval in
 hdfs-default.xml. Contributed by Weiwei Yang.

(cherry picked from commit 186650d21d482e2f3bc4523ae989ebe76081b0e3)
(cherry picked from commit 063f2132ef54436e8a537d437e7a39256a74214f)
(cherry picked from commit 9f9d554edfd83cbd2249c780124a75feebc52ef3)

Change-Id: I10ff500e36f7e9561a021927d7101a91999056de
---
 .../src/main/resources/hdfs-default.xml            |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml b/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
index f9b3635..2d89934 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
@@ -1462,8 +1462,8 @@
   <name>dfs.namenode.stale.datanode.interval</name>
   <value>30000</value>
   <description>
-    Default time interval for marking a datanode as "stale", i.e., if 
-    the namenode has not received heartbeat msg from a datanode for 
+    Default time interval in milliseconds for marking a datanode as "stale",
+    i.e., if the namenode has not received heartbeat msg from a datanode for
     more than this time interval, the datanode will be marked and treated 
     as "stale" by default. The stale interval cannot be too small since 
     otherwise this may cause too frequent change of stale states. 
-- 
1.7.9.5

